sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/BulkUploadActorOperation.java

```diff
@@ -9,7 +9,8 @@
   ORG_BULK_UPLOAD_BACKGROUND_JOB("orgBulkUploadBackground"),
 
   USER_BULK_UPLOAD("userBulkUpload"),
-  USER_BULK_UPLOAD_BACKGROUND_JOB("userBulkUploadBackground");
+  USER_BULK_UPLOAD_BACKGROUND_JOB("userBulkUploadBackground"),
+  USER_BULK_MIGRATION("userBulkMigration");
```

Comment:
This is not direct migrate activity , so other user will be confuse , better to put relative name.